shepherd_bms/include/compute.h

```diff
@@ -2,5 +2,59 @@
 #define COMPUTE_H
 
 #include "datastructs.h"
+#include "nerduino.h"
 
+using namespace std;
+
+class ComputeInterface
+{
+    private:
```

Comment:
This is probably just a lack of my knowledge on the architecture, but are we declaring BMS-wide faults and data here eventually, or is there another higher level that will define that? Stuff like saving the current reading, controlling DCL, etc

Reply:
This class is just meant as an interface for the IO of the compute board, we shouldn't be storing any internal states in it really